src/Microsoft.AspNet.Mvc.Razor/RazorPageActivator.cs

```diff
@@ -87,7 +87,8 @@ private PageActivationInfo CreateViewActivationInfo(Type type)
                 PropertyActivators = PropertyActivator<ViewContext>.GetPropertiesToActivate(
                     type,
                     typeof(RazorInjectAttribute),
-                    CreateActivateInfo)
+                    CreateActivateInfo,
+                    includeNonPublic: true)
```

Comment:
should we change the `InjectChunkVisitor` to spit out a `public` setter for the generated properties?